src/runner.ts

```diff
@@ -36,15 +36,11 @@ const logger = debug('soda-ts:runner')
  *  runner
  * )(query)
  */
-export const createRunner = <T>({
-  appToken,
-  keysCamelCased,
-  ext = 'json'
-}: AuthOpts = {}) => async (query: Query): Promise<T> => {
-  const url = `https://${query.domain}/${query.apiPath}/${query.src}.${ext}`
-
-  logger(`making req to url: ${url}`)
 
```

Comment:
What is this used for?

Reply:
@walleXD That's the original runner created you guys